src/Sql/Sql/ServiceObjective/Cmdlet/GetAzureSqlServerServiceObjective.cs

```diff
@@ -42,21 +47,19 @@ public class GetAzureSqlServerServiceObjective : AzureSqlServerServiceObjectiveC
         /// <returns>A single server</returns>
         protected override IEnumerable<AzureSqlServerServiceObjectiveModel> GetEntity()
         {
-            ICollection<AzureSqlServerServiceObjectiveModel> results = null;
-
-            if (this.MyInvocation.BoundParameters.ContainsKey("ServiceObjectiveName") && !WildcardPattern.ContainsWildcardCharacters(ServiceObjectiveName))
```

Comment:
Why are you implementing your own wildcard support? Please just keep the current implementation.

Reply:
The returned objects don't have "Id" or "ResourceId" properties, so the current implementation doesn't work.

Reply:
Please run the "Get"  if the Service objective does not contain wildcards, as the list could take much longer/in the future have a different set of properties.

Reply:
SQL capabilities is a really weird API. It only has "List". https://docs.microsoft.com/en-us/rest/api/sql/capabilities/listbylocation